solr/solr-ref-guide/src/collections-api.adoc

```diff
@@ -66,7 +96,29 @@ http://localhost:8983/solr/admin/collections?action=SPLITSHARD&collection=collec
 
 Request the status and response of an already submitted <<Asynchronous Calls,Asynchronous Collection API>> (below) call. This call is also used to clear up the stored statuses.
 
-`/admin/collections?action=REQUESTSTATUS&requestid=_request-id_`
+[.dynamic-tabs]
+--
+[example.tab-pane#v1asyncrequeststatus]
+====
+[.tab-label]*V1 API*
+
+[source,bash]
+----
+http://localhost:8983/solr/admin/collections?action=REQUESTSTATUS&requestid=1000
+
+----
+====
+
+[example.tab-pane#v2asyncrequeststatus]
+====
+[.tab-label]*V2 API*
+
+[source,bash]
+----
+curl -X GET http://localhost:8983/api/cluster/command-status/1000 -H 'Content-Type: application/json'
```

Comment:
[0] It's not wrong, and I'm fine w/ it as is.  But you could drop the "Content-type" header here if you wanted to go for concise-ness.

[0] I tested this out and ran into a response that differs a bit from what I expected.

```
➜  solr-9.0.0-SNAPSHOT git:(main) curl -ilk "http://localhost:8983/api/cluster/command-status/1000" -H "Content-type: application/json"                                       
HTTP/1.1 200 OK
Content-Type: application/json;charset=utf-8
Content-Length: 81

{
  "responseHeader":{
    "status":0,
    "QTime":0},
  "availableSubPaths":{}}
```

Suspicious, I tried out a nonexistent request-id and a malformed (spaces, punctuation, etc. in the ID) request-id and got the exact same response as the one above, which was surprising.  Solr should probably return an error on at least the nonexistent case.

Since all my calls, even clearly invalid ones, spat out the same response, I'm wondering if the API itself is even right.  Did you test this out locally?  Am I overlooking something obvious maybe?

I'll prob file a separate JIRA to get to the bottom of this, unless you see something obvious I'm overlooking in my test?

Reply:
Sorry, this is my fault.   Please see SOLR-15276 / #40.   In the commit I tried to flag that this was depending on another PR.   I should maybe have kept things seperate completely, and not wasted your time!   Lesson learned.   Agreed on the non existant case ;-)

Reply:
In other places on GET's we don't have the "Content-type" header, so removing it.  What do you think about on a DELETE?  Same thing?